Sources/Layout/MessagesCollectionViewFlowLayout.swift

```diff
@@ -155,4 +155,70 @@ open class MessagesCollectionViewFlowLayout: UICollectionViewFlowLayout {
         let calculator = cellSizeCalculatorForItem(at: indexPath)
         return calculator.sizeForItem(at: indexPath)
     }
+    
+    /// Set `incomingAvatarSize` of all `MessageSizeCalculator`s
+    open func setMessageIncomingAvatarSize(_ newSize: CGSize) {
```

Comment:
Think these should be `public` and not `open`? They're helper methods. I don't know if we should allow people to override? I don't see any major harm in leaving it but what do you think?

Reply:
No big differences I think, I'll change this to `public`.